src/components/Card/Card.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from "react";
+import bem from "../../utils/bem";
+
+interface CardProps {
+    /** Uh id  */
```

Comment:
haha

Reply:
🤦🏻‍♀️ I saw this and Kristo's comments and was like helen pls

Reply:
Keep it!

Reply:
Crystal will leave comments about how she's feeling in the stories and [I think it's so funny](https://nypl.github.io/nypl-design-system/storybook/storybook-static/react/?path=/story/link--passed-in-link) hahaha